extensions/shorten/src/shorten.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  Action,
+  ActionPanel,
+  Clipboard,
+  Form,
+  Toast,
+  getPreferenceValues,
+  openCommandPreferences,
+  showToast,
+} from "@raycast/api";
+import { isValidUrl } from "../utils/url";
+import { exec } from "child_process";
+import path from "path";
+import { useState } from "react";
+
+type TShortenerPath = {
+  netlifyshortener: string;
+};
+
+type TData = {
+  url: string;
+};
+
+export default function Command() {
+  const [url, setUrl] = useState<string>("");
+  const preferences = getPreferenceValues<TShortenerPath>();
+
+  const netlifyShortenerPath = path.join(preferences.netlifyshortener, "/node_modules/netlify-shortener/dist/index.js");
```

Comment:
[q] Curious why the user needs to clone the Netlify shortener repo locally and set it as a preference -- could you add https://www.npmjs.com/package/netlify-shortener as a dependency of this extension and call it that way?